src/renderer.ts

```diff
@@ -312,12 +345,14 @@ const imageComponentFromRole = (role: Role): FC<BodyImageProps> => {
     }
 }
 
-const render = (format: Format, imageMappings: ImageMappings) =>
+const render = (format: Format, imageMappings: ImageMappings, excludeStyles = false) =>
     (element: BodyElement, key: number): ReactNode => {
     switch (element.kind) {
 
         case ElementKind.Text:
-            return text(element.doc, format);
+            return excludeStyles
+                ? Array.from(element.doc.childNodes).map(plainTextElement)
+                : text(element.doc, format)
```

Comment:
Let me test a few more today, that might be possible.

Reply:
I've seen some interactive atoms that rely on page styles. This is now a bit tricky when we style individual components rather than having page stylesheets.

Reply:
I think if we want to fix that short term for interactives we can just "polyfill" those styles in at the page level.

For interactive atoms that appear on other articles I agree that's a problem. I think best case would be to solve this upstream in the atoms, because relying on legacy styles available in frontend is brittle and probably something we'd like to avoid.

Reply:
yeah, hopefully it's not much more than link colours and line-heights but we may have to add more styles to the `interactiveAtom` component.

We should meet with Sean Clarke before releasing this to production.